.github/workflows/build-test.yml

```diff
@@ -0,0 +1,35 @@
+name: Solid - Build and Test
+
+on: 
+  push:
```

Comment:
It's unfortunate that for version-tagged builds, we will restore and build twice, once in this workflow and once following. Theoretically, if we made these mutually exclusive with some condition here, we would be able to get to restore and build only once. How feasible is this?

(We would also run the unit tests in the other workflow)

Reply:
Yeah, I thought about that but I'm actually OK with the duplication.  The build-and-test workflow does a debug build and then runs unit tests.  The build-installer-and-release workflow does a `Release` build and then builds/signs the installer and creates a release.  So there is some duplication, but not as much as it looks like.  Also, the tag usually is created separately after a commit is pushed, so there is some separation from these workflow runs.